### DISCUSSION TITLE
Using mysql

### DISCUSSION THREAD

**Discussion by maxoghana** [evershopcommerce/evershop#641] [Q&A]
Hello, 

I'm really interested in using ever shop but on my host, I only have postgres v9 😔.

I can also use a more up to date version of MySQL.

Is there a way to run evershop on MySQL ?

**Comment by EmanuelCG** [comment]
Hi, Currently, Evershop only supports PostgreSQL and does not have native support for MySQL. According to the official Evershop documentation and repository, PostgreSQL is the default and recommended database engine for it, and there is no official configuration to run it with MySQL.

If possible, try to upgrade your PostgreSQL to a more recent version, such as 12 or higher, to ensure compatibility with Evershop.
